py3_test_reqs.txt

```diff
@@ -0,0 +1,2 @@
+six==1.10.0
+git+https://github.com/python/mypy.git@5d2dfce8cf6b9a0a7ce8fe8e3cb5866094459f8c
```

Comment:
Put it in tools/ for now, that's where similar stuff is today.

-Tim Abbott (mobile)
On Apr 25, 2016 13:31, "Eklavya Sharma" <notifications@github.com> wrote:

> In py3_test_reqs.txt
> <https://github.com/zulip/zulip/pull/680#discussion_r60983181>:
>
> > @@ -0,0 +1,2 @@
> > +six==1.10.0
> > +git+https://github.com/python/mypy.git@5d2dfce8cf6b9a0a7ce8fe8e3cb5866094459f8c
>
> I'm a bit doubtful about the name py3_test_reqs.txt. Do you think the
> name is good?
>
> You also pointed out that we might have multiple requirements files in the
> future (separate for testing and production and maybe (hopefully not)
> separate for python 2 and python 3). Maybe we can have a directory 'setup/'
> which contains all files needed to set Zulip up, like the multiple
> requirements files, provision.py, zulip_tools.py, Vagrantfile, etc. This
> file belongs there more according to me.
>
> But for now, should I move it to tools/ or should I let it stay where it
> is?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zulip/zulip/pull/680/files/08e339843b9b0e98469937aa256ba1f2ba30e868#r60983181>
>